src/main/java/edu/kit/provideq/toolbox/meta/Problem.java

```diff
@@ -82,6 +84,26 @@ public Mono<Solution<ResultT>> solve() {
         });
   }
 
+  public void estimateBound() {
+    if (this.input == null) {
+      throw new IllegalStateException("Cannot estimate value without input!");
+    }
+
+    var optionalEstimator = this.type.getEstimator();
+    if (optionalEstimator.isEmpty()) {
+      throw new IllegalStateException("Cannot estimate value without an estimator!");
+    }
+    var estimator = optionalEstimator.get();
+
+    long start = System.currentTimeMillis();
+
+    var bound = estimator.apply(this.input);
```

Comment:
Could you rename this variable to make to make it distinct from the attribute name `bound`?